### PR TITLE
First commit

### PR DESCRIPTION
**Pull request by AHPIXI**
Link to my project; https://cheerful-gingersnap-83e89a.netlify.app/

My apologies for handing in this project after deadline. Can't blame anything else than myself. I had problems with github yesterday and couldn't manage to push. It turned out my computer was set to "main" but it was supposed to be "master". Don't know what I've done as I tried to push weeks ago without any problems. Anyhow - will be more prepared for the next project.

Thank you!

Best

Axel

### CODE REVIEW

**Review by AntonellaMorittu — CHANGES_REQUESTED** [review]
Hi Axel, of course totally understandable, well done for handin in your first project! 👏
It looks very good and you used grids and flexboxes, one thing that needs to be reviewed is the responsiveness of the app for smaller sizes (mobiles and tablets), the app needs to feature 1 column for mobile, 2 for tablet and 4 for desktop. Please review these changes to get the pull request approved,almost there! 💪

File: code/style.css [hunk 1]
```diff
@@ -1,4 +1,201 @@
...
+
+@media only screen and (max-width: 1600px) {
+  .box {
+    grid-template-columns: repeat((4, 20%));
+  }
+}
+
+@media only screen and (max-width: 768px) {
+  .box {
+    grid-template-columns: repeat(auto-fit, minmax(400px, 1fr));
```
Comment (AntonellaMorittu) [inline code/style.css:193]: we want to display 2 columns on tablets, so it's better to remove this rule to have full control of our layout. Try using simply grid-template-columns: repeat(2, 1fr)

File: code/style.css [hunk 2]
```diff
@@ -1,4 +1,201 @@
...
+    flex-wrap: wrap;
+  }
+
+  .more-news {
+    min-height: 20vh;
+    margin-bottom: 20px;
+  }
+
+  .box {
+    grid-template-columns: repeat(3, 30%);
```
Comment (AntonellaMorittu) [inline code/style.css:172]: This rule is affecting the tablet layout, be mindful and remember we want to display 2 columns only on tablets

**Review by AntonellaMorittu — CHANGES_REQUESTED** [review]
Hi Axel, please see the comments and apply the changes accordingly. Make sure to test the app to check that the responsiveness is correct for all displays. Thank you 🤓

File: code/style.css (hunk 2 above)
Comment (AHPIXI) [reply code/style.css]: Hi Antonellan, many thanks for your help. I have made some changes and pushed it to github. The site look much better now. 
Best
Axel

File: code/style.css [hunk 3]
```diff
@@ -1,4 +1,182 @@
...
+  }
+}
+
+@media only screen and (max-width: 1600px) {
+  .box {
+    grid-template-columns: repeat((4, 20%));
+  }
+}
+
+@media only screen and (max-width: 768px) {
```
Comment (AntonellaMorittu) [inline code/style.css:172]: please apply this rule to @media only screen and (min-width: 481px) and (max-width: 1024px) this is the range for tablets ❇️

**Review by AntonellaMorittu — CHANGES_REQUESTED** [review]
Please see comment, almost there 👍

File: code/style.css (hunk 3 above)
Comment (AHPIXI) [reply code/style.css]: Thank you. I have now applied that

**Review by AntonellaMorittu — APPROVED** [review]

**Pull request closed without merging**